modules/importLandWeb_BAP_Layers/data/CHECKSUMS.txt

```diff
@@ -1,18 +1 @@
-"file" "checksum" "algorithm"
-"2016-12-1 14.58.57-LANDWEB_DATA_LAYERS.tar.gz" "e2d1afe187defa8f" "xxhash64"
-"CC_1990_100m.tif" "568457e24b460bdf" "xxhash64"
-"CC_1990_FILLED_100m.tif" "d5ccd240611ec0c8" "xxhash64"
-"CC_2000_100m.tif" "84d3afeb783c66db" "xxhash64"
-"CC_2010_100m.tif" "0bbfd926ecbccf45" "xxhash64"
-"HT_1990_100m.tif" "da3d07f1f81eef4f" "xxhash64"
-"HT_1990_FILLED_100m.tif" "1d1bf49817b64433" "xxhash64"
-"HT_2000_100m.tif" "c8463cc6f36381b1" "xxhash64"
-"HT_2010_100m.tif" "20f07accdcfcdbea" "xxhash64"
-"SPP_1990_100m.tif" "f3cdc185605c1be4" "xxhash64"
-"SPP_1990_100m.tif.aux.xml" "4625b130a83bfd1a" "xxhash64"
-"SPP_1990_FILLED_100m.tif" "b5a593ca84732250" "xxhash64"
-"SPP_1990_FILLED_100m.tif.aux.xml" "4625b130a83bfd1a" "xxhash64"
-"SPP_2000_100m.tif" "3451ff4dfb78e71a" "xxhash64"
-"SPP_2000_100m.tif.aux.xml" "4625b130a83bfd1a" "xxhash64"
-"SPP_2010_100m.tif" "f2da64e2a3a45de9" "xxhash64"
-"SPP_2010_100m.tif.aux.xml" "4625b130a83bfd1a" "xxhash64"
+"file" "checksum" "filesize" "algorithm"
```

Comment:
why were all of these removed?

Reply:
I think this was automatically done when i ran CHECKSUMS for all modules. Can you reject this bit?

Reply:
Uhmm, actually it seems that these CHECKSUMS were missing a field (filesize?)